pkg/cosign/tsa_test.go

```diff
@@ -117,3 +117,7 @@ func TestGetTSACertsFromTUF(t *testing.T) {
 	require.NotNil(t, tsaCerts.RootCert)
 	require.Len(t, tsaCerts.RootCert, 1)
 }
+
+func TestGetTSACertsFromLocalTUF(t *testing.T) {
+	_, _ = GetTSACerts(context.Background(), "", GetTufTargets)
```

Comment:
I don't think this test is doing anything, let's just remove it.

Reply:
@haydentherapper Done